powerVCF.psm1

```diff
@@ -347,6 +369,75 @@ Function Start-VCFBackup
 }
 Export-ModuleMember -Function Start-VCFBackup
 
+Function Start-VCFRestore
+{
+  	<#
+    	.SYNOPSIS
+    	Start the SDDC Manager restore
+
+    	.DESCRIPTION
+    	The Start-VCFRestore cmdlet invokes the SDDC Manager restore task
+
+    	.EXAMPLE
+    	PS C:\> Start-VCFRestore -backupFile "/tmp/vcf-backup-sfo-vcf01-sfo-rainpole-io-2020-04-20-14-37-25.tar.gz" -passphrase "VMw@re1!VMw@re1!"
+    	This example shows how to start the SDDC Manager restore
+  	#>
+      Param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$backupFile,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$passphrase
+    )
+
+  	Try {
+        createCloudBuilderHeader
```

Comment:
Should this function be made more generic as this isnt a cloud builder use case?

Reply:
Good catch I meant to go back and change that as I was juggling between the two auth methods. Will look at it now